kolibri/core/device/utils.py

```diff
@@ -486,6 +487,7 @@ def get_device_info(version=DEVICE_INFO_VERSION):
     return info
 
 
+@lru_cache()
```

Comment:
Using `lru_cache` without a `max_size` does not use `lru` capabilities, it's a normal cache without memory limit. This is not a problem in this case because not many dataset_ids will be used to call this function.  Maybe using a `@cache` decorator would result in a clearer code for future devs.

Reply:
Ah, I was misled by our backport, which has a default max_size parameter: https://github.com/learningequality/kolibri/blob/develop/kolibri/utils/lru_cache.py#L54 - we should ensure other uses of lru_cache in the codebase add a max_size parameter as well, as none of them do.

Reply:
Looks like the functools version has a default maxsize too? https://docs.python.org/3/library/functools.html#functools.lru_cache